pages/story/[id].tsx

```diff
@@ -25,9 +25,11 @@ import { get, list } from '../../lib/api/stories'
 import StoryDetail from '../../components/stories/StoryDetail'
 import { storyCite } from '../../components/stories/model'
 import FloatingRibbon, { Button } from '../../components/common/FloatingRibbon'
+import HeadTags from '../../components/common/HeadTags'
+import { storyImage } from '../../components/stories/model'
 
 const shareIconSize = 64
-const contentSize = 150
+const contentSize = 130
```

Comment:
Tightened this up slightly to make it fit within the max ~150 chars for a `description` meta tag. If this is a problem @silent1mezzo, I can adjust.

Reply:
My only concern is it shortens the content for social. Content description just gets cut off if it goes longer so I'd probably leave this at 150 in favour of a better social share

Reply:
Yeah, fair. I'll adjust it for my meta tags needs and leave it alone for social.